### PR TITLE
Add missing definitions of package directories to setup.py

### PR DESCRIPTION
**Pull request by HannesHolste**
See [here](https://github.com/biocore/qiita/issues/1847#issuecomment-222020146).

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -42,11 +42,13 @@
       test_suite='nose.collector',
       packages=['qiita_core',
                 'qiita_db',
+                'qiita_db/handlers',
                 'qiita_db/metadata_template/',
-                'qiita_pet',
+                'qiita_pet/',
```
Comment (antgonza) [inline setup.py]: Why do you need the final slash here but not on the other folders?

Comment (HannesHolste) [inline setup.py]: typo because I was testing exactly that. Turns out it doesn't matter if it has slash or no slash.

Comment (antgonza) [inline setup.py]: OK, should be consistent, just because? No strong feelings about it so up to you ...

**Comment by antgonza** [comment]
👍 once tests pass.

**Pull request merged**